### PR TITLE
Default Confs now in subfolder. No more accidentally sending your password

### PR DESCRIPTION
**Pull request by TeoTwawki**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](https://github.com/project-topaz/topaz/blob/master/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Moved them to a new subdirectory and added the original location to ignore list. Put a read me file in the original location to tell new people how to use the confs. Perhaps a afutture update can auto-copy any "missing" file from the default directory.

Also: adjusted some settings to be modern retail in the defaults. Existing servers will be retaining modified confs, this won't affect them. 

**Left speed settings alone** because while modern retails base has moved from 40 to 50, the database has a mix of the 2 for mobs and NPCs. And I think for players the base for what "0" is should be modified before changing the default setting here. Right now 0 = 40 and we I feel should either make that 50 or make 0 actually be 0 and set the default setting to 50.

### CODE REVIEW

**Review by ibm2431 — APPROVED** [review]

File: conf/default/map.conf [hunk 1]
```diff
@@ -100,8 +100,8 @@ speed_mod: 0
 mob_speed_mod: 0
 
 #Allows you to manipulate the constant multiplier in the skill-up rate formulas, having a potent effect on skill-up rates.
-skillup_chance_multiplier: 2.5
-craft_chance_multiplier: 2.6
+skillup_chance_multiplier: 1.0
```
Comment (kaincenteno) [inline conf/default/map.conf]:
```suggestion
skillup_chance_multiplier: 1.0
craft_chance_multiplier: 1.0
```
Is this the current chance on retail?

Comment (TeoTwawki) [reply conf/default/map.conf]: See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

Comment (TeoTwawki) [reply conf/default/map.conf]: See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

Comment (TeoTwawki) [reply conf/default/map.conf]: See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

Comment (TeoTwawki) [reply conf/default/map.conf]: See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

Comment (TeoTwawki) [reply conf/default/map.conf]: See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

Comment (TeoTwawki) [reply conf/default/map.conf]: Thanks shithub

**Pull request merged**